src/main/java/gregtech/common/metatileentities/multi/MetaTileEntityLargeBoiler.java

```diff
@@ -242,10 +236,12 @@ private double getHeatEfficiencyMultiplier() {
 
     @Override
     protected void updateFormedValid() {
-        if (fuelBurnTicksLeft > 0 && currentTemperature < boilerType.maxTemperature) {
-            --this.fuelBurnTicksLeft;
-            if (getOffsetTimer() % 20 == 0) {
-                this.currentTemperature++;
+        if (fuelBurnTicksLeft > 0) {
+            if (getOffsetTimer() % 2 == 0 || currentTemperature < boilerType.maxTemperature) {
+                --this.fuelBurnTicksLeft;
```

Comment:
Since you changed to `getOffsetTimer() % 2`, don't you need to subtract 2 from the `fuelBurnTicksLeft`, since this is no longer being called every tick, but rather every other tick?